components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/authz/OAuth2AuthzEndpoint.java

```diff
@@ -1917,13 +1922,13 @@ private String getSpTenantDomain(String clientId) throws InvalidRequestException
     private String getLoginTenantDomain(OAuthMessage oAuthMessage, String clientId) throws InvalidRequestException {
 
         if (!IdentityTenantUtil.isTenantedSessionsEnabled()) {
-            return getSpTenantDomain(clientId);
+            return EndpointUtil.getSPTenantDomainFromClientId(oAuthMessage.getClientId());
```

Comment:
any reason to change this line

Reply:
Yes. This method threw a Nullpoint exception for some test data.